src/barbequeue/client.py

```diff
@@ -99,11 +99,13 @@ def wait_for_completion(self, job_id, timeout=None):
                 continue
             return job
 
-    def clear(self):
+    def clear(self, force=False):
         """
         Clear all jobs that have succeeded or failed.
+        :type force: bool
+        :param force: Whether to clear all jobs from the job storage queue, regardless if they've been completed or not.
```

Comment:
made me lookup docstrings - you're absolutely right, and I should fix some syntax errors that I've been adding around the codebase lately.